integration_tests/integration_test.py

```diff
@@ -71,21 +70,27 @@ def setUp(self):
         )
 
         rest_communication.post_entry('projects', {'project_id': self.project_id, 'samples': [self.sample_id]})
-
-        self.dynamic_patches = []
         self._test_success = True
 
     def tearDown(self):
         super().tearDown()
         self._reset_logging()
-        for p in self.dynamic_patches:
-            p.stop()
```

Comment:
What did this do?

Reply:
`self.patches` is now a list not a tuple, i.e. is no longer immutable. This means we can now add extra patches to that, and the superclass calls `stop` on them automatically.